src/components/Header/Header.tsx

```diff
@@ -0,0 +1,45 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+const mainNavItems = [
+  { title: 'Главная', href: '#' },
+  { title: 'Услуги', href: '#' },
+  { title: 'Косметика', href: '#' },
+  { title: 'Блог', href: '#' },
+  { title: 'Франшиза', href: '#' },
+  { title: 'Инвестиции', href: '#' },
+  { title: 'Вакансии', href: '#' },
+  { title: 'Барберы', href: '#' },
+  { title: 'Контакты', href: '#' },
+  { title: 'Филиалы', href: '#' },
+];
+
+const Header = () => {
```

Comment:
set `(): JSX.Element => {...}` - the return type for React Component. No need to import it, it is global thing